### PR TITLE
Adding numbers to whatsnew 3.2

### PR DESCRIPTION
**Pull request by bsipocz**
This PR and #8805 are included in the PR number.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -141,6 +141,7 @@ Dan Foreman-Mackey <foreman.mackey@gmail.com> Dan F-M <foreman.mackey@gmail.com>
 Johnny Greco <jgreco@astro.princeton.edu> Johnny <jgreco@astro.princeton.edu>
 Zé Vinicius <jvmirca@gmail.com> mirca <jvmirca@gmail.com>
 Michael Seifert <michaelseifert04@yahoo.de> --system <michaelseifert04@yahoo.de>
```
Comment (MSeifert04) [inline .mailmap]: Ah, seems like I tried to update my system gitconfig and messed up 😰

File: docs/whatsnew/3.2.rst [hunk 2]
```diff
@@ -217,3 +219,80 @@ Full change log
 
 To see a detailed list of all changes in version v3.2, including changes in
 API, please see the :ref:`changelog`.
+
+Contributors to the v3.1 release
```
Comment (MSeifert04) [inline docs/whatsnew/3.2.rst]: Shouldn't this be x3.2?

File: .mailmap (hunk 1 above)
Comment (bsipocz) [reply .mailmap]: water under the bridge, that was in the previous release 😅

File: docs/whatsnew/3.2.rst (hunk 2 above)
Comment (bsipocz) [reply docs/whatsnew/3.2.rst]: thanks, it should indeed!

**Comment by pllim** [comment]
Needs a rebase, LGTM otherwise.

**Pull request merged**